src/main/webapp/index.jsp

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<title>canvas_courses_report</title>
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<meta name="description" content="">
+<meta name="author" content="">
+<meta http-equiv="Cache-control" content="no-cache">
+<meta http-equiv="X-UA-Compatible" content="IE=9">
+<script src="assets/vendor/jquery/jquery-1.11.0.min.js" type="text/javascript"></script>
+<script src="assets/js/analytics.js" type="text/javascript"></script>
```

Comment:
In an app with a lot of UI might consider adding these later, as it stands it does not matter much.